runtime/doc/quickref.txt

```diff
@@ -572,7 +572,6 @@ In Insert or Command-line mode:
 
 |:set|		:se[t]			  show all modified options
 |:set|		:se[t] all		  show all options
-|:set|		:se[t] termcap		  Do nothing. (|terminfo|)
```

Comment:
Don't need to mention this in more places than the  `:help :set-termcap` entry in `vim_diff.txt`.